### DISCUSSION TITLE
🥰 [FEATURE] label.layout 能不能像G2 一样可以定制

### DISCUSSION THREAD

**Discussion by HOUSHENGREN** [antvis/G2Plot#3558] [Q&A] [answered]
### 🥰 Features description [Please make everyone to understand it]
laybel.layout只能使用现有的，无法定制。
### 🏞 What problem does this feature solve
文字重合，我想自定义layout位置，目前不支持
![image](https://user-images.githubusercontent.com/48871516/236180695-f6a44439-affd-42d4-bdfa-90aeb090edd0.png)

### 🧐 What does the proposed API look like
实现像G2一样可以定制layout =》[api](https://antv-g2.gitee.io/zh/docs/api/advanced/register-label#%E8%87%AA%E5%AE%9A%E4%B9%89-label-%E5%B8%83%E5%B1%80%E5%87%BD%E6%95%B0)

**Comment by hustcc** [comment, marked as answer]
可以的。

```ts
import { G2 } from '@antv/g2plot';

G2. registerGeometryLabelLayout('xxx', 'yyy')
```
